grtext/grfont.cpp

```diff
@@ -999,8 +999,8 @@ void grfont_XlateColorGrayChar(int bmp_handle, int x, int y, int index, tFontFil
   sptr = (ushort *)ft->char_data[index];
   rowsize_w = bm_rowsize(bmp_handle, 0) / 2;
 
-  // height = SET_MIN(h, ddsfObj.h);
-  // width = SET_MIN(w, ddsfObj.w);
+  // height = std::min(h, ddsfObj.h);
+  // width = std::min(w, ddsfObj.w);
```

Comment:
?

Reply:
it's best to fix commented out code too otherwise it loses all meaning because the macro no longer exists.